Cargo.toml

```diff
@@ -22,13 +22,13 @@ required-features = ["async-tokio"]
 [dependencies]
 bitflags = "1.0"
 libc = "0.2"
-nix = "0.14"
-tokio = { version = "0.2", features = ["io-driver", "rt-threaded", "macros"], optional = true }
+nix = "0.20"
+tokio = { version = "1", features = ["io-util", "rt-multi-thread", "macros"], optional = true }
```

Comment:
i don' think `rt-mult-thread` should be defined here but in an application using this (otherwise they won't be able to choose the runtime which will be weird), other than that lgtm.

Reply:
Alright, I have removed it now.